src/api/ChatsAPI.ts

```diff
@@ -0,0 +1,65 @@
+import BaseAPI from './BaseAPI';
+import {Options} from '../utils/httptransport';
+
+export interface Message {
+  chat_id: string,
+  content: string,
+  file: string,
+  id: number,
+  is_read: boolean,
+  time: string,
```

Comment:
будет удобнее работать со временем типа Date
```suggestion
  time: Date,
```